### ISSUE TITLE
General Issue: A special case discussion when derive the confirmed best overall response by derive_param_confirmed_bor()

### ISSUE THREAD

**Issue by sukiegao** [pharmaverse/admiralonco#244]
### Background Information

We encountered a case during actual use, as below screenshot:

![image](https://github.com/pharmaverse/admiralonco/assets/105427846/0dc34755-0232-451d-8139-49381c8f94c0)

The first row is the visit timepoint, the second row is tumor overall response result in each visit, the third row is ADT, and the fourth row is ADY. Confirmed Best Overall Response (BOR) is derived based on these 3 results. 

According to RECIST 1.1, if PR followed by PD,  and the time between PR and the first dose date meet the minimum duration of SD which is 6 weeks,  then confirmed BOR result is SD, otherwise it's PD.  

In this case,  the ADY of first PR  is 37 days, which is less than 6 weeks(study also specifies that a 6-week interval is required), so confirmed BOR result should be PD, but this R package will select the date of the first occurrence of PD (2017-10-27) as confirmed BOR’s ADT. But in my understanding, the date of PR (2017-09-15) should be select as confirmed BOR’s ADT.

As an example of a comparison, screenshot as below:

![image](https://github.com/pharmaverse/admiralonco/assets/105427846/bdedfa91-7ae0-4001-a539-3f1ec55676fa)

In this case, confirmed BOR is SD, and PR’s date will be selected as confirmed BOR's ADT.

I’m not sure it’s a bug in package or my incorrect understanding. 

To sum up, my question is: In the first case, Which date should we choose as confirmed BOR’s ADT, the date of PR? or the date of first PD?

### Definition of Done

Not sure the answer, just raise this special case to discuss. 
It‘s expected that someone who encounters this special case can give some advice.

**Comment by bundfussr** [comment]
@sukiegao , I would consider 2017-10-27 as date of BOR in the first case because on 2017-09-15 definitely no PD occurred. The Roche standard handles this case in the same way.

**Comment by bundfussr** [comment]
@pharmaverse/admiralonco , any thoughts?

**Comment by sukiegao** [comment]
Thanks @bundfussr @rossfarrugia ！Thought about this again, I think it makes sense that it's over- conservative if we assume it a PD. Since tumor progression is a process, if the 6 weeks evaluation interval is not met and there is indeed no sign of PD, it's also unreasonable for us to select 2017-09-15 as date of BOR .

**Comment by rossfarrugia** [comment]
thanks - will close this issue
